README_tmpl.md

```diff
@@ -0,0 +1,301 @@
+the gloss to indicate that the first is the agent-like argument and the second is the
+patient-like argument.
+
+[example 19](cldf/examples.csv#cldf:19)
+
+
+### Rule 5: Person and number labels
+
+Person and number are not separated by a period when they cooccur in this order.
+E.g.
+
+[example 20](cldf/examples.csv#cldf:20)
+
+
+### Rule 5A. (Optional)
+
+Number and gender markers are very frequent in some languages, especially when
+combined with person. Several authors therefore use non-capitalized shortened
+abbreviations without a period. If this option is adopted, then the gloss
+in (21b) is used.
+
+[example 21a](cldf/examples.csv#cldf:21a)
+
+[example 21b](cldf/examples.csv#cldf:21b)
+
+
+### Rule 6: Non-overt elements
+
+If the morpheme-by-morpheme gloss contains an element that does not correspond
+to an overt element in the example, it can be enclosed in square brackets (as in 22a). An
+obvious alternative is to include an overt "∅" in the object-language text, which is
+separated by a hyphen like an overt element (as in 22b).
+
+[example 22a](cldf/examples.csv#cldf:22a)
+
+[example 22b](cldf/examples.csv#cldf:22b)
+
+
+### Rule 7: Inherent categories
+
+Inherent, non-overt categories such as gender may be indicated in the gloss, but a
+special boundary symbol, the round parenthesis, is used. E.g.
+
+[example 23](cldf/examples.csv#cldf:23)
+
+
+### Rule 8: Bipartite elements
+
+Grammatical or lexical elements that consist of two parts which are treated as
+distinct morphological entities (e.g. bipartite stems such as Lakhota na-xʔu̧ 'hear')
+may be treated in two different ways:
+
+(i) The gloss may simply be repeated:
+
+[example 24](cldf/examples.csv#cldf:24)
+
+(ii) One of the two parts may be represented by a special label such as STEM:
+
+[example 25](cldf/examples.csv#cldf:25)
+
+Circumfixes are "bipartite affixes" and can be treated in the same way, e.g.
+
+[example 26a](cldf/examples.csv#cldf:26a)
+
+[example 26b](cldf/examples.csv#cldf:26b)
+
+
+### Rule 9: Infixes
+
+Infixes are enclosed by angle brackets, and so is the object-language counterpart in
+the gloss.
+
+[example 27](cldf/examples.csv#cldf:27)
+
+[example 28](cldf/examples.csv#cldf:28)
+
+Infixes are generally easily identifiable as left-peripheral (as in 27) or as right-
+peripheral (as in 28), and this determines the position of the gloss corresponding to
+the infix with respect to the gloss of the stem. If the infix is not clearly peripheral,
+some other basis for linearizing the gloss has to be found.
+
+
+### Rule 10: Reduplication
+
+Reduplication is treated similarly to affixation, but with a tilde (instead of an
+ordinary hyphen) connecting the copied element to the stem.
+
+[example 29](cldf/examples.csv#cldf:29)
+
+[example 30](cldf/examples.csv#cldf:30)
+
+[example 31](cldf/examples.csv#cldf:31)
+
+
+## Appendix: List of Standard Abbreviations
+
+[List of abbreviations](cldf/abbreviations.csv#cldf:__all__)
```

Comment:
@fmatter rendering data via templates also works for non-standard tables included with a CLDF dataset.